modules/python/python-plugin.c

```diff
@@ -40,6 +40,7 @@
 #include "reloc.h"
 
 #include <stdlib.h>
+#include <string.h>
```

Comment:
Is this header needed?

Reply:
It's in the wrong place, I'm moving it to `compat-python-v{2,3}.c`. Thanks.

Reply:
Oh too late, I'll fix it in another PR then :smile: